pkg/operator/operator.go

```diff
@@ -165,6 +167,7 @@ func New(
 		mcpInformer.Informer(),
 		proxyInformer.Informer(),
 		oseKubeAPIInformer.Informer(),
+		maoSecretInformer.Informer(),
```

Comment:
Jerry this is where we use the actual informer adding it to the event handler so we get a resync every time something changes (which calls syncMachineConfigPools)